packages/valory/skills/register_termination_abci/__init__.py

```diff
@@ -21,9 +21,9 @@
 This module contains a dummy ABCI skill in which the agents register and then pick up the termination signal.
 
 It is created only for testing purposes for the termination feature.
-"""
+"""  # pragma: nocover
 
-from aea.configurations.base import PublicId
+from aea.configurations.base import PublicId  # pragma: nocover
```

Comment:
why two?

It should be enough to place it on line 25 for the whole file

Reply:
It does not work with placing the no cover tag at the top, I tried it as the first thing